src/components/ui/UserPicture.vue

```diff
@@ -76,7 +77,7 @@ export default {
       ) {
         this.src = await generateIdenticon(this.avatar.username, this.size);
       } else {
-        this.src = `${this.avatar.picture}?size=${this.slot}`;
+        this.src = `${avatarUrl(this.avatar.picture)}?size=${this.slot}`;
```

Comment:
Seems like if we are going to the trouble of building a ```avatarUrl()``` function, we might want to include the query string parameters since that's also part of the url.

Reply:
:+1: that